api/build.gradle

```diff
@@ -639,10 +639,10 @@ def db_host = System.getenv("DB_HOST")
 def db_port = System.getenv("DB_PORT")
 
 def commandLineSpringProperties = [
-        'spring.datasource.driver-class-name': 'com.mysql.jdbc.Driver',
+        'spring.datasource.driver-class-name': 'com.mysql.cj.jdbc.Driver',
         // Our database isn't configured to allow outside connections (which is good),
         // so our connections are either local or through the encrypted proxy.
-        'spring.datasource.url'              : "jdbc:mysql://${db_host}:${db_port}/workbench?useSSL=false",
+        'spring.datasource.url'              : "jdbc:mysql://${db_host}:${db_port}/workbench?useSSL=false&allowPublicKeyRetrieval=true",
```

Comment:
other solutions: https://community.atlassian.com/t5/Confluence-questions/MySQL-Public-Key-Retrieval-is-not-allowed/qaq-p/778956